README.md

```diff
@@ -430,15 +389,113 @@ When **running the CLI from your CI pipeline** you will need to export two varia
 Go to your Settings page in Checkly and grab a fresh API key from [the API keys tab](https://app.checklyhq.com/settings/user/api-keys) and your
 Account ID from the [Account settings tab](https://app.checklyhq.com/settings/account/general).
 
-# Constructs API
+# Using the Constructs API
+
+All resources you can create and manage using the Checkly CLI are derived from "constructs". These constructs are just 
+[TypeScript classes](https://github.com/checkly/checkly-cli/tree/main/package/src/constructs) like `ApiCheck` in `api-check.ts` and 
+`SlackAlertChannel` in `slack-alert-channel.ts`.
 
-*This will document all the properties of the various constructs based on TSDoc annotations*
+You can use standard JS/TS programming to use these constructs to create the monitoring setup of your
+choice. Loops, variables, if-statements, file imports, extensions etc.
 
 ## Project
 
-## Check
 
-### ApiCheck
+## Checks
+
+The CLI currently supports two Check types: API Checks and Browser Checks. All checks share common properties, like:
+
+- `name` : A human readable name for your Check.
+- `frequency`: How often to run you check in minutes, i.e. `60` for every hour.
+- `locations`: An array of location codes where to run your checks, i.e. `['us-east-1', 'eu-west-1]`.
+- `activated`: A boolean value if your check is activated or not.
+- `muted`: A boolean value if alert notifications from your check are muted, i.e. not sent out.
+- `doubleCheck`: A boolean value if Checkly should double check on failure.
+- `tags`: An array of tags to help you organize your checks, i.e. `['product', 'api']`
+- `runtimeId`: The ID of which [runtime](https://www.checklyhq.com/docs/runtimes/specs/) to use for this check.
+
+Note that most properties have sane default values and do not need to be specified.
+
+### API Checks
+
+API checks are a good fit for monitoring typical HTTP based endpoints like REST APIs and GraphQL APIs, but can also be
+used for form encoded payloads. [Read more about API checks in our docs](https://www.checklyhq.com/docs/api-checks/)
+
+- It defines the basic check properties like `name`, `activated` etc.
```

Comment:
I think that the same example appears earlier in the readme under the "Creating an API Check" section. I think that it's a bit confusing having almost the same example twice. 

To keep the readme size manageable, I think that it makes sense to have one section that gives a getting started example, then another section that is more of a reference.

Reply:
@clample I cleaned up the first example, it's much simpler now.